cluster/operator-push.sh

```diff
@@ -28,6 +28,8 @@ else
     push_registry=127.0.0.1:$(./cluster/cli.sh ports registry | tr -d '\r')
 fi
 
+export MONITORING_NAMESPACE=${MONITORING_NAMESPACE:-openshift-monitoring}
```

Comment:
why do we need this on every `make operator-push` and not simply on the monitoring lane?

Reply:
removed